driver/aws/rds.py

```diff
@@ -95,3 +96,61 @@ def get_db_type(db_instance_identifier: str, client: RDSClient) -> str:
     if db_type == "aurora":  # for aurora mysql 5.6
         db_type = "aurora_mysql"
     return db_type
+
+
+def get_db_parameter_group_name(db_instance_identifier: str, client: RDSClient) -> str:
+    """
+    Get database parameter group name
+    """
+    db_parameter_group_name = ""
+    instance_info = get_db_instance_info(db_instance_identifier, client)
+    parameter_groups = instance_info.get("DBParameterGroups", "")
```

Comment:
It's more idiomatic to just do `.get("key")`, which makes the variable `None` if the key is nonexistent.